trunk/NDHMS/Data_Rec/module_namelist.F

```diff
@@ -735,7 +771,33 @@ subroutine rt_nlst_check(nlst)
       call hydro_stop("Compound channel option not available for diffusive wave routing. ")
    end if
 
-
+   if(nlst%reservoir_persistence) then
+      if(len(trim(nlst%reservoir_persistence_parameter_file)) .eq. 0) then
+         call hydro_stop('hydro.namelist ERROR: You MUST specify a reservoir_persistence_parameter_file for &
+         inputs to persistence type reservoirs.')
+      endif
+      if(len(trim(nlst%reservoir_timeslice_path)) .eq. 0) then
+         call hydro_stop('hydro.namelist ERROR: You MUST specify a reservoir_timeslice_path for &
+         reservoir persistence capability.')
+      endif
+      if(len(trim(nlst%reservoir_persistence_parameter_file)) .ne. 0) then
+        inquire(file=trim(nlst%reservoir_persistence_parameter_file),exist=fileExists)
+        if (.not. fileExists) call hydro_stop('hydro.namelist ERROR: reservoir_persistence_parameter_file not found.')
+      endif
+      ! Can leave both blocks below commented out because the reservoir module_read_timeslice_data.F will
```

Comment:
can this be removed?

Reply:
I assume you just mean the commented out section. Yes.

Reply:
yes, comment is attached to the first line.... though it is not obvious.

Reply:
@jdmattern-noaa fixing

Reply:
@jdmattern-noaa fixing